py/selenium/webdriver/remote/webelement.py

```diff
@@ -313,25 +317,15 @@ def send_keys(self, *value):
             # file_input.send_keys(os.path.abspath("path/to/profilepic.gif"))
 
         """
+        ignore_files = kwargs.get('ignore_files', False)
         # transfer file to another machine only if remote driver is used
         # the same behaviour as for java binding
-        if self.parent._is_remote:
+        if self.parent._is_remote and not ignore_files:
             local_file = self.parent.file_detector.is_local_file(*value)
```

Comment:
did you look at trying to set the 'file_detector'? this was provided to allow exactly what you're trying to do.

for example if you wanted to disable it:

    from selenium.webdriver.remote import file_detector
    driver.file_detector = file_detector.UselessFileDetector()

    #and then to set it back to default behavior
    driver.file_detector = file_detector.LocalFileDetector()